sklearn/model_selection/tests/test_search.py

```diff
@@ -1172,6 +1173,11 @@ def test_search_cv_timing():
             assert_true(search.cv_results_[key][0] == 0.0)
             assert_true(np.all(search.cv_results_[key] < 1))
 
+        assert_true(hasattr(search, "refit_time_"))
+        assert_true(isinstance(search.refit_time_, float))
+        if sys.version_info[0] >= 3:
+            assert_greater(search.refit_time_, 0)
```

Comment:
why not >= as for the other parameters?

Reply:
Just did this - why didn't I realize this before... Please excuse the extra commits.